pp/go/relabeler/head/catalog/catalog.go

```diff
@@ -13,8 +13,7 @@ import (
 )
 
 const (
-	logFileName    = "head.log"
-	MaxLogFileSize = 32 * 1024
+	DefaultMaxLogFileSize = 1 << 22
```

Comment:
```suggestion
	DefaultMaxLogFileSize = 4 << 20 // 4 MB
```
expression `<< 10`, `<< 20` is understandable as KiB and MiB. But `<< 22` is not so clear. It is better write only binary prefix in short form.